tools/xtce_generation/telemetry_generator.py

```diff
@@ -246,9 +246,9 @@ def create_remaining_parameters(self, parameter_set):
 
             description = Et.SubElement(parameter, "xtce:LongDescription")
 
-            if row["longDescription"] is None and row["shortDescription"] is None:
+            if row.get("longDescription") is None and row.get("shortDescription") is None:
```

Comment:
This branching reads a little odd to me with all the checks. Perhaps something like this, or is it too confusing at first glance?

```python
description.text = row.get("shortDescription") or row.get("longDescription") or ""
```
Which I think will check them in the order you want.

Reply:
Yeah, this is much more readable and cleaner. will add that. thanks!